documentation/modules/exploit/linux/http/kaltura_unserialize_cookie_rce.md

```diff
@@ -0,0 +1,46 @@
+## Description 
+
+The getUserzoneCookie function in Kaltura before 13.2.0 uses a hardcoded cookie secret to validate cookie signatures, which allows remote attackers to bypass an intended protection mechanism and consequently conduct PHP object injection attacks and execute arbitrary PHP code via a crafted userzone cookie. 
+ 
+
+## Vulnerable Application
+
+ This module exploits a remote code execution within the Kaltura(<=13.1.0) via a cookie deserialization.
+ Vulnerability reference- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14143.
```

Comment:
Is there no install instructions? or a .deb or something else you could help here to get someone to install?  Even just referencing the install docs from Kaltura is good start.